code/modules/nano/nanoui.dm

```diff
@@ -86,7 +88,7 @@ nanoui is used to open and update nano browser uis
 	add_template("main", ntemplate_filename)
 
 	if (ntitle)
-		title = sanitize(strip_improper(ntitle))
+		title = sanitize(ntitle)
```

Comment:
That's important proc. It's fix crashing for some clients what have open nanoUI windows.